Makefile

```diff
@@ -121,6 +128,10 @@ docker-stability:
 	cd integration/stability && ./bind_mount_linux.sh
 	cd integration/stability && ./hypervisor_stability_kill_test.sh
 
+podman: ginkgo
+	./ginkgo -failFast -v -focus "${FOCUS}" -skip "${SKIP}" \
```

Comment:
mmm but `SKIP` is only for docker, isn't it?

https://github.com/kata-containers/tests/blob/4ae15b1ccd6e04866555d577a9154e1e712d60c3/arch/aarch64-options.mk#L12-L13

Reply:
@devimc yes, I will actually later use SKIP when I used e2e tests as some of them are failing but instead of docker filter will be podman